README.rst

```diff
@@ -13,6 +13,7 @@ Running commands from the edx-platform directory will default to loading the
 configuration at ``./conf/locale/config.yaml``. You can specify a different
 configuration file with the ``--config`` argument.
 
+* ``i18n_tool detect``
```

Comment:
All three names look good to me (detect, change, diff) because we detect changes/differences. Pick one you think best describes the process I'm not sure which one to use.
@clintonb described the process in the comment section https://openedx.atlassian.net/browse/SOL-2053